lib/logstash/inputs/eventlog.rb

```diff
@@ -89,6 +93,8 @@ def run(queue)
 
       end # while
 
+    rescue LogStash::ShutdownSignal
+
```

Comment:
After looking at other plugin we are already doing this in the zeromq, kafka output and the twitter input.